securedrop-log/debian/securedrop-log.install

```diff
@@ -0,0 +1 @@
+securedrop.Log etc/qubes-rpc/securedrop.Log
```

Comment:
so `securedrop.Log` isn't yet in `securedrop-log` (though it's described in the README how to create) - we should add a file over there called `securedrop.Log` containing just `/usr/sbin/securedrop-log` and then this should work

Reply:
Wait, I thought I pushed that in a PR last night. Let me check

Reply:
Yup, I forgot to create the PR (it was after midnight I guess) https://github.com/freedomofpress/securedrop-log/pull/3